integration/compareDissecCentralized.spec.js

```diff
@@ -0,0 +1,101 @@
+const { execSync } = require('child_process')
+const {
+  default: CozyClient,
+  createClientInteractive,
+  Q
+} = require('cozy-client')
+
+const { populateCentralized } = require('../scripts/populateCentralized')
+const { dissecLearning } = require('./helpers/dissecLearning')
+const { localLearning } = require('./helpers/localLearning')
+const { BANK_DOCTYPE } = require('../src/doctypes/bank')
+const { JOBS_DOCTYPE } = require('../src/doctypes/jobs')
+
+describe('Compares the performance of a centralized learning vs the DISSEC one', () => {
+  const defaultTimeout = 150000
+
+  const uri = 'http://test1.localhost:8080'
+  let client
+  let token
+  let operations = []
+  let validationSet = []
+  let categories = []
+  let cutoffDate
+
+  beforeAll(async () => {
+    await populateCentralized()
```

Comment:
`populateCentralized` does not sound very good while it will actually populate 10 instances. Maybe `populateInstances` would be better?
It would also be nice to be able to provide: 
- A `baseUrl` , e.g. `test`, so it will populate `test1.cozy.localhost`, `test2.cozy.localhost`, etc.
- The number of instances to populate

Reply:
The name `Centralized` is due to how it splits data between instances (90 samples for 1 instances, 10 samples for 9 other instances). It's different from `populate`, which gives an equal amount of data to each instance.

Also, `populateCentralized` can only handle 10 instances for now, hence the absence of parameters.

I can rename `populate` into `populateInstances` and `populateCentralized` into `populateInstancesCentralized`?

Reply:
I think the `centralized` terminology is a bit confusing here.

But actually, the 2 scripts could be merged into one `populate`? And depending on the params, the data could be populated uniformly or not. But this could be left for a future PR

Reply:
And in any case, we should add some documentation in the README as the code grows in complexity ;)